Source/Scene/PointCloud3DTileContent.js

```diff
@@ -448,9 +460,16 @@ define([
         content._hasColors = defined(colors);
         content._hasNormals = defined(normals);
         content._hasBatchIds = defined(batchIds);
+
+        // Compute an approximation for base resolution in case it isn't given
+        // Assume a uniform distribution of points in the bounding sphere around the tile.
+        var radius = content._tile.contentBoundingVolume.boundingSphere.radius;
+        var sphereVolume = (4.0 / 3.0) * CesiumMath.PI * radius * radius * radius;
```

Comment:
Will the sphere always be available?

Is it better to use a box - if it was provided - since it would be a tighter fit around the tile than a sphere computed around the box?

Add a function or property getter to `BoundingSphere` to compute the volume instead of hardcoding it here.

Reply:
`TileBoundingVolume` currently specifies that all implementations should have a `boundingSphere` instead of a box, and in practice this also seems to be decent. I think it's also better to overestimate the geometric error here to avoid underattenuation. I'll add a comment for the reasoning.

Reply:
Sounds good.